evap/staff/templates/staff_semester_view_evaluation.html

```diff
@@ -78,6 +78,9 @@
         {% endif %}
     </div>
     {% include 'evaluation_badges.html' with mode='manager' %}
+    {% if evaluation.course.evaluations.count > 1 %}
```

Comment:
If performance could be an issue here we might want to think about annotating that instead of manually traversing the relations here -- @karyon?

Reply:
annotation done